vllm/distributed/device_communicators/custom_all_reduce_utils.py

```diff
@@ -169,9 +183,15 @@ def gpu_p2p_access_check(i: int, j: int) -> bool:
         #  enter this block to calculate the cache
         logger.info("generating GPU P2P access cache in %s", path)
         cache = {}
+        batch_is = []
+        batch_js = []
         for _i in range(num_dev):
             for _j in range(num_dev):
-                cache[f"{_i}->{_j}"] = can_actually_p2p(_i, _j)
+                batch_is.append(_i)
+                batch_js.append(_j)
```

Comment:
Can consider simplify to
```
batch_is = [i // num_dev for i in range(num_dev * num_dev)]
batch_js = [i % num_dev for i in range(num_dev * num_dev)]
```
Also it might be clearer to name them `batch_src` and `batch_dst`.

Reply:
fixed in [bc12163](https://github.com/vllm-project/vllm/pull/5528/commits/bc12163ad196582cabef5d07135e5a667766b089)